source/iface/integer-functions.rst

```diff
@@ -7,3 +7,855 @@
 *****************
 Integer functions
 *****************
+
+The function descriptions in this section use the term
+generic integer type to represent the following types:
```

Comment:
```suggestion
`generic integer type` to represent the following types:
```

Reply:
Like the the other pages in this section, can you please update "generic integer type" throughout this page to be in italics, for consistency and clarity?

Reply:
Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/163/commits/ac505b5425f863c51032de809b166de03f05707d

Reply:
I think the italics change should be through the entire page - the term appears many times.  Other pages in this section format as italics throughout the entire page.  Can you please update all instances to be in italics?

Reply:
Added with : https://github.com/KhronosGroup/SYCL_Reference/pull/163/commits/9749ff9a3b2c07f30bbebf9f7440e5f6f38c7fdc